preliz/distributions/exgaussian.py

```diff
@@ -101,6 +102,14 @@ def entropy(self):
         logpdf = self.logpdf(x_values)
         return -np.trapz(np.exp(logpdf) * logpdf, x_values)
 
+    def mode(self):
+        tau = self.nu
```

Comment:
why renamming  the variable? I get nan for some combinations of parameters like pz.ExGaussian(0, 1, 10)